bin/visualize

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
This is the main script that allows users to display dependencies for both a single directory and by walking a directory. It also supports the use of a -s flag which will give dependency info for a single directory vs walking a directory.

Reply:
What does the actual script do?

- Create a networkx Directed Graph to store dependencies.
- Checks if there is a cycle ( circular dependency )
- Finds the source nodes of the graph (nodes that have no dependencies and can be potentially run solo at any time)
- Creates a dependency list and prints it out in human readable output.
- This script will also check for any directories which have not been set up to be added to the graph (no dependency information) and add them to a list of directories to be run after the graph.